templates/list.html

```diff
@@ -3,10 +3,10 @@
 	<div class="col-xs-10 col-xs-offset-1">
 		<div class="list-group">
 			{% if "none" in pastes %}
-			<a href="#" class="list-group-item">No pastes yet</a>
+				<a href="/new" class="list-group-item">No pastes yet</a>
```

Comment:
Do we need the tab there? I think the Jinja2 code won't be in the final HTML file and therefore the tab may not be needed. On the other hand this makes it clearer to code. What do you think?

The link to `/new` however is a good idea. I think we can keep it.